include/InstrumentTrack.h

```diff
@@ -221,6 +222,9 @@ class LMMS_EXPORT InstrumentTrack : public Track, public MidiEventProcessor
 
 	void autoAssignMidiDevice( bool );
 
+	BoolModel *m_midiCCEnable;
+	FloatModel *m_midiCCModel[MIDI_CC_MAX_CONTROLLERS];
```

Comment:
I think you can use `MidiControllerCount` from `Midi.h` and remove the `MidiCCRackView.h` include.